rasterio/warp.py

```diff
@@ -229,6 +229,9 @@ def reproject(source, destination, src_transform=None, gcps=None,
         remain in all areas not covered by the reprojected source.
         Defaults to the nodata value of the destination image (if set),
         the value of src_nodata, or 0 (GDAL default).
+    dst_resolution: tuple (x resolution, y resolution) or float, optional
+        Target resolution, in units of target coordinate reference
+        system.
     src_alpha : int, optional
         Index of a band to use as the alpha band when warping.
     dst_alpha : int, optional
```

Comment:
@snowman2 can you add a "Returns" section of the docstring explaining that this method returns an `array, Affine` tuple?

Reply:
Sure, just updated the returns section.